solr/core/src/java/org/apache/solr/handler/component/ShardResponse.java

```diff
@@ -81,6 +81,10 @@ void setResponseCode(int rspCode) {
     this.rspCode = rspCode;
   }
 
+  public int getRspCode() {
+    return rspCode;
+  }
+
```

Comment:
👍 surprising that this wasn't already accessible! Seems pretty general-purpose useful.